src/main/java/com/exactpro/th2/common/schema/message/impl/rabbitmq/connection/ConnectionManager.java

```diff
@@ -441,6 +449,7 @@ public void confirm() throws IOException {
                     }
                 }, cancelCallback), configuration);
 
+        holder.isSubscribed = true;
```

Comment:
Can we encapsulate this into the holder class? I think it would be better to have just a `subscribed` method in holder class